modules/core/src/test/java/org/apache/ignite/internal/processors/security/IgniteSecurityProcessorTest.java

```diff
@@ -46,6 +48,11 @@ public void testSwitchToDenyAllPolicyIfNodeNotFoundInDiscoCache() {
         when(ctx.discovery()).thenReturn(mock(GridDiscoveryManager.class));
         when(ctx.log(any(Class.class))).thenReturn(mock(IgniteLogger.class));
 
+        MarshallerContextImpl mockMarshallerContext = mock(MarshallerContextImpl.class);
+        when(mockMarshallerContext.jdkMarshaller()).thenReturn(new JdkMarshaller());
```

Comment:
TEST_JDK_MARSHALLER

Reply:
This class is not a subclass of GridCommonAbstractTest.